tp_estimator.js

```diff
@@ -1,75 +1,111 @@
 (function(){
+  //declare all the variables and the column headers.
```

Comment:
When you get to the point of a 'final product', you should keep in mind that a professional setting will not want all of your comments left in the code. It's unnecessary as long as you've named variables and written/formatted your code well, which you've done. 

Just keep it in mind...